functions/Copy-SqlLogin.ps1

```diff
@@ -152,6 +152,12 @@ Limitations: Does not support Application Roles yet
 					continue
 				}
 				
+				if($destserver.LoginMode -ne "Mixed")
+				{ 
+					Write-Output "$Destination does not have Mixed Mode enabled. Enable this after the migration."
```

Comment:
>Login never appears to be migrated anyway 

This seems to be a bug. I don't have an environment to test that on. I can fix the other stuff however.